pymeasure/instruments/ni/daqmx.py

```diff
@@ -77,7 +77,7 @@ def setup_analog_voltage_in(self, channelList, numSamples, sampleRate=10000, sca
                                 DAQmx_Val_Rising,DAQmx_Val_FiniteSamps,
                                 uInt64(self.numSamples)));
 
-    def setupAnalogVoltageOut(self, channel=0):
+    def setup_analog_voltage_out(self, channel=0):
```

Comment:
I understand the rational for modifying the style on these to match the rest of the package. The reason these differ is because the underlying DAQmx software uses the camel case format. I think it is reasonable to change the names as you did, since the instrument classes are only included for convenience.

Reply:
I had changed this before actually trying to use the DAQ drivers as written. Unfortunately, I couldn't get the daqmx file included with PyMeasure to control my instruments. I ended up using PyDAQmx instead, which seemed to work. I hadn't included those changes with this pull request. After working with DAQ a bit more, I'm actually of the opinion that DAQ should be treated as an adapter of some sort and instruments made which use DAQ as their only adapter, but this is probably a larger discussion which should happen somewhere else.

Reply:
I agree this is a topic for further discussion. I will include these changes for now.